### PR TITLE
Fix server mingw build

### PR DESCRIPTION
**Pull request by manisandro**

### CODE REVIEW

File: src/server/qgsserver.cpp [hunk 1]
```diff
@@ -290,7 +290,7 @@ bool QgsServer::init()
 
 void QgsServer::putenv( const QString &var, const QString &val )
 {
-#ifdef _MSC_VER
+#ifdef Q_OS_WIN32
```
Comment (nyalldawson) [inline src/server/qgsserver.cpp]: this function should be using qputenv -- that's already got all the cross platform handling done for us!

**Comment by nyalldawson** [comment]
@pblottiere ?

**Comment by pblottiere** [comment]
Hi @manisandro,

Sorry for the latency :/.

It looks good to me, but can you use `qunsetenv()` when the value is empty instead? Otherwise we'll have a different behavior between platforms (empty vs deleted).

Thanks :+1:

**Comment by manisandro** [comment]
Sure, done, thanks for the feedback.

**Comment by pblottiere** [comment]
Thanks @manisandro

**Pull request merged**